### PR TITLE
adding test to OpenSCAP content entity in Airgun

### PR DESCRIPTION
**Pull request by tstrych**
@oshtaier  or @abalakh  please review.

### CODE REVIEW

File: tests/foreman/ui_airgun/test_oscapcontent.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        session.oscapcontent.upload_new({
+            'title': title,
+            'scap_file': oscap_content_path,
+        })
+        assert session.oscapcontent.search(title)[0]['Title'] == title
+
+
+def test_positive_delete(session):
+    title = gen_string('alpha')
+    oscap_content_path = settings.oscap.content_path
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_oscapcontent.py]: you can move that out into module fixture

**Comment by tstrych** [comment]
Providing test results:
```============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/tstrych/Documents/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 3 items2018-08-07 18:15:53 - conftest - DEBUG - Collected 3 test cases


test_oscapcontent.py                                                  [100%]

============================== 0 tests deselected ==============================
========================== 3 passed in 97.84 seconds ===========================```

**Comment by tstrych** [comment]
test_results:
```============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/tstrych/Documents/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 3 items2018-08-08 16:04:44 - conftest - DEBUG - Collected 3 test cases


test_oscapcontent.py                                                  [100%]

============================== 0 tests deselected ==============================```

File: tests/foreman/ui_airgun/test_oscapcontent.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    :id: 9870555d-0b60-41ab-a481-81d4d3f78fec
+
+    :Steps:
+
+        1. Create an openscap content.
+        2. Provide all the appropriate parameters.
+        3. Update the openscap content, here the Org.
+
+    :expectedresults: Whether creating  content for OpenScap is successful.
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_oscapcontent.py]: `Whether updating`?

**Comment by tstrych** [comment]
test results:
```
============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/tstrych/Documents/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 3 items2018-08-28 16:33:52 - conftest - DEBUG - Collected 3 test cases


test_oscapcontent.py                                                  [100%]

============================== 0 tests deselected ==============================
========================== 3 passed in 127.31 seconds ==========================
```

File: tests/foreman/ui_airgun/test_oscapcontent.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        session.oscapcontent.create({
+            'file_upload.title': title,
+            'file_upload.scap_file': oscap_content_path,
+        })
+        session.oscapcontent.update(title, {
+            'organizations.resources.assigned': [org.name]
+        })
+        oscap_val = session.oscapcontent.read(title)
+        assert org.name in oscap_val['organizations']['resources']['assigned']
+        assert oscap_content_path.rsplit(
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_oscapcontent.py]: I would propose to expand `test_positive_create` with `read` functionality and move that verification (with any additional verifications if possible) there

**Comment by tstrych** [comment]
test results:
```============================= test session starts ==============================
platform linux -- Python 3.6.6, pytest-3.4.0, py-1.5.3, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/tstrych/Documents/robottelo, inifile:
plugins: wait-for-1.0.9, services-1.2.1, mock-1.6.3
collected 3 items2018-08-29 21:55:52 - conftest - DEBUG - Collected 3 test cases


test_oscapcontent.py                                                  [100%]

============================== 0 tests deselected ==============================
========================== 3 passed in 170.83 seconds ==========================
```

File: tests/foreman/ui_airgun/test_oscapcontent.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture, tier2
+
+
+@fixture(scope='module')
+def oscap_content_path():
+    return settings.oscap.content_path
+
+
+def test_positive_create(session, oscap_content_path):
```
Comment (oshtaier) [inline tests/foreman/ui_airgun/test_oscapcontent.py]: after last change, we can treat that test case as tier2, so if you work with that file in future, just add the tag and docstring, not a blocker atm

**Pull request merged**